sensors/lidar/lidar.go

```diff
@@ -0,0 +1,53 @@
+// Package lidar implements the Lidar sensor
+package lidar
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/pointcloud"
+	"go.viam.com/rdk/registry"
+
+	"go.viam.com/slam/config"
+	"go.viam.com/slam/sensors"
+)
+
+// Lidar represents a LIDAR sensor.
+type Lidar struct {
+	Name         string
+	lidar        camera.Camera
+	DataRateMsec int
+}
+
+// New creates a new Lidar sensor based on the sensor definition and the service config.
+func New(ctx context.Context, deps registry.Dependencies, sensor sensors.Sensor, svcConfig *config.AttrConfig) (Lidar, error) {
+	name, err := sensor.GetName(svcConfig)
+	if err != nil {
+		return Lidar{}, err
+	}
+
+	newLidar, err := camera.FromDependencies(deps, name)
+	if err != nil {
+		return Lidar{}, errors.Wrapf(err, "error getting camera %v for slam service", name)
```

Comment:
Do we want to start distinguishing between a camera and a sensor (like lidar)?

Reply:
Good point. I'll edit the comment to make this clearer.